scripts/zones/Beadeaux/mobs/Ge_Dha_Evileye.lua

```diff
@@ -10,13 +10,17 @@ require("scripts/zones/Beadeaux/MobIDs");
 -- onMobSpawn
 -----------------------------------
 
+function onMobSpawn(mob)
+    mob:setLocalVar("benedictionTrigger", math.random(10,50));
+end;
 
 -----------------------------------
 -- onMobFight Action
 -----------------------------------
 
 function onMobFight( mob, target )
-    if (mob:getHPP() < 30 and mob:getLocalVar("benediction") == 0) then
+ local hpTrigger = mob:getLocalVar("benedictionTrigger");
```

Comment:
:)

missed an indent

Reply:
you made me laugh waffle all over my computer screen!!!